spec/fixtures/052/valid_example.xml

```diff
@@ -14,6 +14,10 @@
     <Id>0352C5220131227110203</Id>
     <ElctrncSeqNb>130000005</ElctrncSeqNb>
     <CreDtTm>2013-12-27T11:01:46.0+01:00</CreDtTm>
+    <FrToDt>
+      <FrDtTm>2009-01-01T00:00:00:0+01:00</FrDtTm>
```

Comment:
This does not look like a valid value
Shouldn't this be `2009-01-01T00:00:00.0+01:00`